docs/app/Examples/elements/Rail/Variations/RailSizeExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Image, Grid, Rail, Segment } from 'stardust'
+
+const RailSizeExample = () => (
+  <Grid className='three column'>
+    <Grid.Column className='centered'>
+      <Segment>
+        <Image src='http://semantic-ui.com/images/wireframe/paragraph.png' />
+
+        <Rail size='tiny' position='left'>
+          <Segment>Left Rail Content</Segment>
+        </Rail>
+
+        <Rail size='tiny' position='right'>
+          <Segment>Right Rail Content</Segment>
+        </Rail>
+      </Segment>
+    </Grid.Column>
+  </Grid>
+)
```

Comment:
It looks like the size sets `font-size Xrem;`.  The Semantic docs show incorrect usage of the reail sizes.  This does not work with the Segment child because the Segment uses `rem` as well, which is the absolute base font size.  It should either use `em` in this case, so it inherits the Rail font-size and scales, or we should just remove the Segment.

I vote we remove the Segment and also show all the sizes:

```js
import React, { PropTypes } from 'react'
import { Image, Grid, Rail, Segment } from 'stardust'

const Wrapper = ({ children }) => (
  <Grid className='three column center aligned'>
    <Grid.Column className='centered'>
      <Segment>
        <Image src='http://semantic-ui.com/images/wireframe/paragraph.png' />
        {children}
      </Segment>
    </Grid.Column>
  </Grid>
)

Wrapper.propTypes = {
  children: PropTypes.node,
}

const RailSizeExample = () => (
  <div>
    <Wrapper>
      <Rail size='tiny' position='left'>Tiny Left Rail</Rail>
      <Rail size='tiny' position='right'>Tiny Right Rail</Rail>
    </Wrapper>
    <Wrapper>
      <Rail size='large' position='left'>Large Left Rail</Rail>
      <Rail size='large' position='right'>Large Right Rail</Rail>
    </Wrapper>
    <Wrapper>
      <Rail size='huge' position='left'>Huge Left Rail</Rail>
      <Rail size='huge' position='right'>Huge Right Rail</Rail>
    </Wrapper>
    <Wrapper>
      <Rail size='massive' position='left'>Massive Left Rail</Rail>
      <Rail size='massive' position='right'>Massive Right Rail</Rail>
    </Wrapper>
  </div>
)

export default RailSizeExample
```

Which renders:

![image](https://cloud.githubusercontent.com/assets/5067638/16853010/61842db0-49bf-11e6-9900-30c67756fb2b.png)